llvm/test/CodeGen/PowerPC/pr92543.ll

```diff
@@ -0,0 +1,15 @@
+; RUN: llc -ppc-asm-full-reg-names -o - %s| FileCheck %s
```

Comment:
the file name should contain the bug issue number 62372, not the patch number.(a little confusing : ))